config/nimbus.yaml

```
@@ -33,3 +33,31 @@ features:
         value: { "enabled": true }
       - channel: production
         value: { "enabled": false }
+  automatic-removal-csat-survey:
+    description: Show the CSAT survey for Plus users that have automatically removed broker results
+    variables:
+      enabled:
+        description: If the feature is enabled
+        type: Boolean
+        default: false
+    defaults:
+      - channel: local
+        value: { "enabled": true }
+      - channel: staging
+        value: { "enabled": true }
+      - channel: production
+        value: { "enabled": false }
+  latest-scan-date-csat-survey:
+    description: Show the CSAT survey for users that are enrolled in the `latest-scan-date` experiment
```

```suggestion
  last-scan-date-csat-survey:
    description: Show the CSAT survey for users that are enrolled in the `last-scan-date` experiment
```

The experiment is named `last-scan-date`, do we want it to be the same here for consistency?

---

Yes a 💯% — renamed in [306872c](https://github.com/mozilla/blurts-server/pull/4611/commits/306872c977ead1a9cc7c694ad667811fbe99c073).